GliaWidgets/CallVisualizer/VisitorCode/VisitorCodeViewModel.swift

```diff
@@ -0,0 +1,78 @@
+import UIKit
+import SalemoveSDK
+
+extension CallVisualizer {
+    class VisitorCodeViewModel {
+        let presentation: CallVisualizer.Presentation
+        let visitorCodeAlertStyle: AlertStyle
+        var environment: Environment
+        var delegate: (Delegate) -> Void
+        var timer: FoundationBased.Timer?
+        var theme: Theme
+        var receivedVistorCode: VisitorCode? {
+            didSet {
+                notifyPropsUpdated()
+                scheduleVisitorCodeRefresh()
+            }
+        }
+
+        init(
+            presentation: CallVisualizer.Presentation,
+            visitorCodeAlertStyle: AlertStyle,
+            environment: Environment,
+            theme: Theme,
+            delegate: @escaping (Delegate) -> Void
+        ) {
+            self.presentation = presentation
+            self.visitorCodeAlertStyle = visitorCodeAlertStyle
+            self.environment = environment
+            self.delegate = delegate
+            self.theme = theme
+        }
+
+        func props() -> VisitorCodeViewController.Props {
```

Comment:
please correct me if I wrong, however, we have team decision of using verb in function name. @gersonnoboa what do you think, will an verb make this function name better?

Reply:
fixed [676c74c](https://github.com/salemove/ios-sdk-widgets/pull/448/commits/676c74ceb781899d4ca4fa8a8ab684c8a1837308)

Reply:
To be honest I don't fully remember. I think we have that only for things that return booleans, `should`, `is`, etc. For ones that don't return anything, it's  `doSomething`, but for ones that return something, it can be `something`, as we normally don't use `getSomething`. So in this case, `props()` would be better than `getProps()`.